external/webclient/webclient.c

```diff
@@ -1005,81 +1008,93 @@ static pthread_addr_t wget_base(void *arg)
 		goto errout;
 	}
 
-	buf_len = 0;
-	while (sndlen > 0) {
-		if (param->tls) {
-			ret = mbedtls_ssl_write(&(client_tls->tls_ssl),
-									(const unsigned char *)param->buffer + buf_len,
-									sndlen);
-		} else {
-			ret = send(sockfd, param->buffer + buf_len, sndlen, 0);
-		}
-		if (ret < 1) {
-			ndbg("ERROR: send failed: %d\n", ret);
-			goto errout;
-		} else {
-			sndlen -= ret;
-			buf_len += ret;
-			ndbg("SEND SUCCESS: send %d bytes\n", ret);
-		}
+	send_len = sndlen;
```

Comment:
I think this is unnecessary

Reply:
This is required to send the same length buffer for KEEP_ALIVE_REQUEST_COUNT